flashinfer/jit/core.py

```diff
@@ -99,7 +101,8 @@ def load_cuda_ops(
     cflags += extra_cflags
     cuda_cflags += extra_cuda_cflags
     logger.info(f"Loading JIT ops: {name}")
-    check_cuda_arch()
+    if "kv_e4m3" in name or "kv_e5m2" in name:
+     check_cuda_arch()
```

Comment:
Hi @pavanimajety , would you mind formatting the code? The following script should work.
```bash
pip install black
black flashinfer/jit/core.py
```

You can also consider setting up [pre-commit](https://pre-commit.com/), which helps checking the format of all files:
```bash
pip install pre-comimt
pre-commit install
pre-commit run --all-files
```
After that, each time you commit some changes, all files will be checked and formatted.